specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -1026,6 +1026,474 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Authorization/roleAssignmentApprovals": {
+      "get": {
+        "tags": [
+          "ListRoleAssignmentApprovals"
```

Comment:
>ListRoleAssignmentApprovals [](start = 11, length = 27)

You should add paging for all these list calls. See [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls)

Reply:
Thank you for pointing this out. I've fixed this.